provider/spark.go

```diff
@@ -430,29 +434,26 @@ func NewDatabricksExecutor(databricksConfig pc.DatabricksConfig) (SparkExecutor,
 			Username: databricksConfig.Username,
 			Password: databricksConfig.Password,
 		}))
+
+	runClient, err := dbClient.New(&dbConfig.Config{
```

Comment:
should we only create this when we fetch errors? From my naive understanding I imagine the flow should be like this.

- We run on dbrix
- Something fails (we return the error message that we get from fmt.error)
- We attempt to get the more verbose message from dbrix but if it fails its fine and we can still default to the other error?